Update README.md (#50)
mirego/kmp-boilerplate: README.md

## Usage

There are 2 ways to use this boilerplate:
1. From the `main` branch: This branch is the "lean" version, it does not contain any opinionated

>    libraries or tools. It is a good starting point if you want to build your own boilerplate.

Reviewer: ```suggestion
   libraries or tools. It is a good starting point if you want a fresh starting point or to even build your own boilerplate.
```

Reviewer: Whoops j'ai mal lu... 🤦‍♂️

2. From the `main-full` branch: This branch contains all the opinionated libraries and tools
   described in the [Preferred libraries](#preferred-libraries) section. It is a good starting

>    point if you want to start building your app right away.

Reviewer: ```suggestion
   point if you want to quickly start building your app the Mirego way.
```

### With GitHub template

> 1. Make sure you choose the correct branch

Reviewer: ```suggestion
1. Pick your preferred branch (`main` or `main-full`)
```

2. Click on the [**Use this template**](https://github.com/mirego/kmp-boilerplate/generate)
   button to create a new repository
3. Clone your newly created project (`git clone https://github.com/you/repo.git`)
4. Run the boilerplate setup script (`./boilerplate-setup.sh`)
5. Commit the changes (`git commit -a -m "Rename kmp-boilerplate parts"`)

### Without GitHub template

1. Clone this project (`git clone https://github.com/mirego/kmp-boilerplate.git`)

> 2. Make sure you choose the correct branch (`git checkout main` or `git checkout main-full`)

Reviewer: ```suggestion
2. Pick your preferred branch (`git checkout main` or `git checkout main-full`)
```
